From 3bc22a9c790f7c617d76bd7a0a02dedf663759ad Mon Sep 17 00:00:00 2001
From: Mesih Kilinc <mesihkilinc@gmail.com>
Date: Mon, 7 Oct 2019 09:08:15 +0300
Subject: [PATCH] qtbase: Fix plug_path for static builds

On static builds, Makefiles's LIBS has wrong path for plugins.
This caused by missing SYSROOT path on front of the plugin paths.

Using $$[QT_INSTALL_PLUGINS] instead of $$[QT_INSTALL_PLUGINS/get]
seams to fix the problem

Signed-off-by: Mesih Kilinc <mesihkilinc@gmail.com>
---
 mkspecs/features/qt.prf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mkspecs/features/qt.prf b/mkspecs/features/qt.prf
index 89f4946c..46923d22 100644
--- a/mkspecs/features/qt.prf
+++ b/mkspecs/features/qt.prf
@@ -148,7 +148,7 @@ import_plugins:qtConfig(static) {
             plug_name = $$QMAKE_PREFIX_STATICLIB$${plug}$$qtPlatformTargetSuffix().$$QMAKE_EXTENSION_STATICLIB
             plug_path = $$eval(QT_PLUGIN.$${plug}.PATH)
             isEmpty(plug_path): \
-                plug_path = $$[QT_INSTALL_PLUGINS/get]
+                plug_path = $$[QT_INSTALL_PLUGINS]
             LIBS += $$plug_path/$$plug_type/$$plug_name
         } else {
             LIBS += -l$${plug}$$qtPlatformTargetSuffix()
-- 
2.23.0

